tests/test_gdppc.py

```diff
@@ -1,59 +1,325 @@
-import os
+"""
+Various unit, smoke, and integration tests for impactcommon.exogenous_economy.gdppc
+"""
 import pytest
-if not os.getenv("IMPERICS_SHAREDDIR"):
-    pytest.skip(
-        "skipping test as no IMPERICS_SHAREDDIR environment variable set",
-        allow_module_level=True
-    )
+import numpy as np
+import pandas as pd
 from impactcommon.exogenous_economy import gdppc
```

Comment:
Can you add back in the "real data" tests, for when we have the shared dir available?

Reply:
With `69ecf951` the old tests are now in `tests/test_gdppc_externaldata.py`. They remain in their own module so that they can be marked and then skipped if external data is not available.

Reply:
That works. Thanks.